src/utils/requests/BillRequests.js

```diff
@@ -0,0 +1,19 @@
+const {URL} = require("../../config");
+/**
+ * @function getBill
+ * @description Function that calls the /bills route to return the bills associated to the user. With bearer token.
+ * @param {String} token
+ */
+export const getBill = async (token) => {
+    const response = await fetch(`${URL}/bills`, { 
+        method: "GET",
+        headers: {
+            "Content-Type": "application/json",
+            "Authorization": `Bearer ${token}`,
+        }
+    });
+
+    return await response.json();
```

Comment:
Also, I believe this is missing handling for errors similar to the PR above

Reply:
its ok to leave it as is, since the error handler is called in the component

Otherwise, he'd have to do this here:

```js
try {
...
} catch (err) {
   throw err;
}
```

for the frontend to respond to error properly (i.e. error prompt). which is quite redundant

Reply:
long as this is functin is kept as a promise, im ok

Reply:
still gotta have tests doe